pysatNASA/tests/test_methods_cdaweb.py

```diff
@@ -49,7 +49,7 @@ def test_load_with_empty_file_list(self):
 
```

Comment:
Recommend adding a `@pytest.mark.parameterize(pandas_format, [True, False])` to this test.  I also think you need to add a `pysat.Meta()` to the `cdw.load_xarray` function return on line 369.

Also recommend adding a test for `cdw.load_pandas` with a bad filename that will trigger the 'unable to load' logger warning and cover line 274 (return empty data and meta).

Reply:
Updated. Note that the file needs to exist for this warning to trigger. Added an empty file to load.

Just this once, i threw in an extra unit test for another uncovered warning.